apps/graphql-playground-v2/package.json

```
@@ -5,8 +5,8 @@
   "dependencies": {
     "@contentful/app-sdk": "^4.23.1",
     "@contentful/f36-components": "^4.54.0",
+    "@contentful/f36-layout-alpha": "npm:@contentful/f36-layout@5.0.0-alpha.18",
```

when we no longer have double-layouts included we can just import the layout component directly on its alpha version and don't need to name it alpha. less cleanup after releasing v5 :)

```suggestion
    "@contentful/f36-layout": "5.0.0-alpha.18",
```